api-lib/frames/HelpFrame.tsx

```diff
@@ -75,7 +91,7 @@ export const HelpFrame: Frame = {
   // TODO: change this
   buttons: [
     {
-      title: 'Use on Farcaster',
+      title: 'Try @givebot',
       action: 'link',
       target:
         'https://warpcast.com/~/compose?text=@givebot @singer did some nice work today&embeds[]=https://farcaster.xyz',
```

Comment:
@wingfeatherwave do we want any embeds or not? if not, lets remov ethis farcaster link